src/PKSim.UI/Views/PopulationAnalyses/PopulationAnalysisStatisticsSelectionView.cs

```diff
@@ -56,11 +54,10 @@ public override void InitializeBinding()
          _gridViewBinder.Changed += NotifyViewChanged;
       }
 
-      private RepositoryItem editRepository(StatisticalAggregation statisticalAggregation)
+      private RepositoryItem editRepository(StatisticalAggregation statisticalAggregation) => new RepositoryItemTextEdit
```

Comment:
This is change in DEvExpress a few version. Before we could use the same instance and change property of the actual instance. (which was kind of weird). Now we need to create a new instance